packages/shared/src/components/Attachments/Attachments.tsx

```diff
@@ -0,0 +1,216 @@
+                                                    key={attachmentItem.id}
+                                                    item={attachmentItem}
+                                                    onAttachmentDelete={() => onAttachmentDelete(attachmentItem)}
+                                                    onAttachmentReplaceWithBrowse={() =>
+                                                        onReplaceAttachmentWithBrowse(attachmentItem)
+                                                    }
+                                                    onAttachmentReplaceWithUpload={(uploadedAsset: Asset) =>
+                                                        onReplaceAttachmentWithUpload(attachmentItem, uploadedAsset)
+                                                    }
+                                                />
+                                            ))}
+                                        </div>
+                                    </SortableContext>
+                                    <DragOverlay>
+                                        {draggedAttachment ? (
+                                            <AttachmentItem
+                                                isEditing={isEditing}
+                                                key={draggedAssetId}
+                                                designTokens={designTokens}
+                                                item={draggedAttachment}
+                                                onAttachmentDelete={() => onAttachmentDelete(draggedAttachment)}
+                                                onAttachmentReplaceWithBrowse={() =>
+                                                    onReplaceAttachmentWithBrowse(draggedAttachment)
+                                                }
+                                                onAttachmentReplaceWithUpload={(uploadedAsset: Asset) =>
+                                                    onReplaceAttachmentWithUpload(draggedAttachment, uploadedAsset)
+                                                }
+                                            />
+                                        ) : null}
+                                    </DragOverlay>
+                                </DndContext>
+                            )}
+                            {isEditing && (
+                                <div className="tw-px-5 tw-py-3">
+                                    <div className="tw-font-body tw-font-medium tw-text-text tw-text-s tw-my-4">
+                                        Add attachments
+                                    </div>
+                                    <AssetInput
+                                        isLoading={isUploadLoading}
+                                        onMultiAssetClick={() => {
+                                            /* This enables multiple file inputs */
+                                        }}
```

Comment:
remove?

Reply:
Due to how the assetinput is implemented if I remove this then selecting multiple files is disabled.